### DISCUSSION TITLE
我可以帮忙在crowdin上完成国际化的中文翻译

### DISCUSSION THREAD

**Discussion by gov360** [PeterDing/chord#70] [Ideas]
请在上面创建项目后，把地址回复到下面。[官网链接](https://zh.crowdin.com/profile/shareuser/machine-translationl)

**Comment by PeterDing** [comment]
抱歉，现在正在做将集成 spotify 到 chord 的开发。国际化内容还没开始。

**Reply by gov360** [reply]
好吧ヽ( ￣д￣;)ノ，我会持续关注
